packages/foundry/tests/replaceTokens.test.ts

```diff
@@ -0,0 +1,101 @@
+import * as fs from "node:fs/promises";
+import * as glob from "glob";
+import { replaceTokens } from "../src/replaceTokens";
+
+jest.mock("node:fs/promises");
+jest.mock("glob");
```

Comment:
Is it possible to test this without using any mock?

I very rarely need to use mocks to test frontend code.

The best pattern to test this function is probably snapshot testing: https://jestjs.io/docs/snapshot-testing